### ISSUE TITLE
New page auto create in Vue2 version

### ISSUE THREAD

**Issue by elvishuges** [motla/vue-document-editor#18]
The function to create new page, in Vue 2 dont work.

When im writing a text and in overflow the a4 page size. the editor dont create a new page automatcaly.

**Comment by motla** [comment]
Hi! Thanks for reporting.
Please provide an example or a github repo so I can test your code and find the bug.

**Comment by elvishuges** [comment]
can you send me a codepen using vue2 version and runnning the vue-document-editor ?

**Comment by motla** [comment]
Come on man, you can do it 🙂 
You just have to take the static code example in the README.md file:
```html
<html>
<head>
  <script src="https://cdn.jsdelivr.net/npm/vue@3/dist/vue.global.prod.js"></script>
  <script src="https://cdn.jsdelivr.net/npm/vue-document-editor@2/dist/VueDocumentEditor.umd.min.js"></script>
  <link href="https://cdn.jsdelivr.net/npm/vue-document-editor@2/dist/VueDocumentEditor.css" rel="stylesheet">
</head>
<body>
  <div id="app">
    <div style="font-family: Avenir, sans-serif">
      <vue-document-editor v-model:content="content" />
    </div>
  </div>
  <script>
  const app = Vue.createApp({
    components: { VueDocumentEditor },
    data () {
      return { 
        content: ["<h1>Hello!</h1>Fill this page with text and new pages will be created as it overflows."]
      }
    }
  }).mount('#app');
  </script>
</body>
</html>
```
Just copy and paste it.

**Comment by elvishuges** [comment]
yes. the code you shared works. but take a look when i use in my project. the text overflow and the new page is not generated:
![image](https://user-images.githubusercontent.com/12242481/195735641-41e58ad6-2615-4fb5-bc1e-c821f81dfc08.png)

**Comment by motla** [comment]
I can't deduce anything from a screenshot... I'm not a god (yet)! Again please provide an example or a github repo so I can test your code and find the bug. It's the only way I can help you.

**Comment by elvishuges** [comment]
Hello, Motla. i created a project using vue2 version and installed your vue-document-editor.
Please download and install this project and see how the vue-document editor behaves. 
1 There is a black line around the editor.
2 please , check the used scss and sass version and tell me if it is ok
3 the editor star with text centered.
4 the editor behaves very diferent whe compared to the live version.

**Comment by motla** [comment]
I think you didn't pushed your latest code to your repo, because I can't see vue-document-editor. I did `npm install` then `npm run serve` it takes me to the Vue2 "hello world" new project default page.

**Comment by elvishuges** [comment]
Sorry. try again please.  i updated the repo. Sorry again

**Comment by motla** [comment]
Hello!
Thanks for your report.
Apparently it had to do with CSS scoping.
I just released v1.3.0 for Vue2 and v2.1.0 for Vue3 which should correct your problem.
Please let me know this if you still have issues.
